exchanges/binance/binance_wrapper.go

```diff
@@ -491,9 +485,9 @@ func (b *Binance) UpdateTickers(ctx context.Context, a asset.Item) error {
 		}
 
 		for y := range tick {
-			cp, err := currency.NewPairFromString(tick[y].Symbol)
+			cp, err := enabledPairs.DeriveFrom(tick[y].Symbol)
 			if err != nil {
-				return err
+				continue
```

Comment:
Why do you treat an error from USDTMarginedFutures differently from CoinMarginedFutures?